message.go

```diff
@@ -47,15 +47,6 @@ func makeMessageId(id string, def string) string {
 	return id
 }
 
-// Returns the time value passed as first argument, unless it's the zero-value,
-// in that case the default value passed as second argument is returned.
-func makeTimestamp(t time.Time, def time.Time) time.Time {
```

Comment:
Why did you remove this function? Seems like it's not related to the underlying bug

Reply:
It was left unused after the bug was fixed. It was just used to set OriginalTimestamp to a default if it wasn't already defined. Actually I should probably re-add that to allow submitting historical events but still with clock skew correction

Reply:
Right, yeah I think let's keep the `m.OriginalTimestamp = makeTimestamp(m.OriginalTimestamp, ts)` lines, so we're only changing what we need to

Reply:
[404997c](https://github.com/cloudquery/analytics-go/pull/1/commits/404997ca0c26840c82bc92bc9718e453c5fa59cc)